recipe/meta.yaml

```diff
@@ -1,45 +1,31 @@
-{% set version = "0.1" %}
-{% set jupyterlab_version = "0.32.0" %}
-{% set sha256 = "489c6ba7df0bb8946bc47c19587e6ed35df82e3df7da0f6e8b17061c30a816e1" %}
+{% set version = "0.2" %}
 
 package:
   name: jupyterlab_delux
   version: {{ version }}
-
+  
 source:
-    fn: jupyterlab-{{ jupyterlab_version }}.tar.gz
-    url: https://pypi.io/packages/source/j/jupyterlab/jupyterlab-{{ jupyterlab_version }}.tar.gz
-    sha256: {{ sha256 }}
+  path: ../
+
 build:
   noarch: python
   number: 0
   entry_points:
-    - jupyter-lab = jupyterlab.labapp:main
-    - jupyter-labextension = jupyterlab.labextensions:main
-    - jupyter-labhub = jupyterlab.labhubapp:main
+    - jupyterlab_delux = jupyterlab_delux:main
```

Comment:
This is so cool!
Do you think it would be possible to provide a `delux` specific entry point for the `labextension` command? So that there would be a nice way to install additional extensions without changing environment variables or specifying the app dir?

Something like:
```
delux-labextension install ...
delux-labextension list
```

I'm also thinking I'd rather call the entry point to launch jupyterlab `delux-lab` instead of `jupyterlab_delux`.

Reply:
That's a great idea. I look into it. 

And your naming suggestion is also better.